Makefile

```diff
@@ -23,3 +23,24 @@ infra: ## 🚀 Deploy the API Ops Infrastructure
 extract-env: ## ⚙️ Extract the environment variables from Terraform
 	@echo -e "\e[34m$@\e[0m" || true
 	@./scripts/json-to-env.sh < infrastructure/terraform_output.json > infrastructure/terraform.env
+
+extract-dev: ## 📤 Export the development API Management instance
```

Comment:
We can communicate this in the README/comments but I'd like to ensure the user knows this is for local extraction. The extractor pipeline extracts the configuration to the /apimartifacts folder by default, so the fact that this extracts to the /extract folder (if I'm reading this correctly) might trip some up. In the final instructions that have the different make commands, I'd like to ensure the user knows this is to a special folder of sorts.

Reply:
Agreed - great comment - just changed the description and moved it to a folder called "temp" to signify that this is really demoware